schemas/common/page.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/page",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Page Object",
+  "type": "object",
+  "description": "Page object contains paging information for related resource when resource result is paginated.",
+  "definitions": {
+    "page": {
+      "properties": {
+        "orderBy": {
+          "title": "Order by property",
+          "type": "string",
+          "description": "The property by which this page is ordered.\n"
+        },
+        "order": {
+          "title": "Sort order",
+          "type": "string",
+          "pattern": "^(asc|desc)$",
+          "description": "Sort order, either \"asc\" or \"desc\".\n"
+        },
+        "start": {
+          "title": "First value",
+          "type": "string",
+          "description": "The first value, in sorted order, of the orderby property on this page.\n"
```

Comment:
Make sure `orderBy` is in camelcase, even in descriptions. Use backticks to make it stand out. (the `\n` at the end is not needed)

Reply:
Question: is `orderBy` only allowed for `string` values? If no, the type of this property might be `anyOf` string, number, boolean (at least)

Reply:
`orderBy` is a comma-separated list of property names, optionally prefixed with `+` or `-`. Hence it is a `string` always.

Reply:
`orderBy` is always a `string`, that much is clear. The question is if ordering is only allowed for properties that have string values (which does not seem to make sense). In that case, the `start` and `next` properties should be able to have other values, too.

Reply:
What does `+` and `-` mean? All of that needs to go into the description string.